### ISSUE TITLE
Customizable Rollups: After initial push upgrade, the default CRLP Settings record is not being created

### ISSUE THREAD

**Issue by force2b** [SalesforceFoundation/Cumulus#3200]
The legacy rollup jobs check to see if CRLP is enabled before running. So, when the schedule kicks off multiple of those jobs at the same time all of them are checking CustomizableRollupsSettings object and when the record doesn't exist they're all trying to insert a record. The first one wins, the rest get this error. It should be a one time error though, and should only happen if two processes running at exactly the same time try to insert the default record.

```
Upsert failed. First exception on row 0; first error: DUPLICATE_VALUE, duplicate value found: SetupOwnerId duplicates value on record with id: 00DB00000003Izu: []

Stacktrace available at the time of saving the record - Class.npsp.UTIL_CustomSettingsFacade.getOrgCustomizableRollupSettings: line 855, column 1
Class.npsp.UTIL_CustomSettingsFacade.getCustomizableRollupSettings: line 839, column 1
Class.npsp.CRLP_Rollup_SVC: line 46, column 1
Class.npsp.RLLP_OppRollup_UTIL.setupRollupVars: line 243, column 1
Class.npsp.RLLP_OppRollup_UTIL.<init>: line 182, column 1
Class.npsp.RLLP_OppSoftCreditRollup_BATCH.<init>: line 40, column 1
Class.npsp.RLLP_OppSoftCreditRollup_BATCH.execute: line 47, column 1
```

**Comment by force2b** [comment]
**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x8000000AXSQYAA5" target="_blank">W-025017</a>

**Comment by mrbelvedere** [comment]
Included in beta release 3.130 (Beta 3)

**Comment by mrbelvedere** [comment]
Included in production release 3.130
